index.js

```diff
@@ -78,12 +78,27 @@ function dockerPostgres(dockerOpts) {
     // set the timeout of the hook long enough for the container to start
     this.timeout(10000);
     return new Promise(function (resolve, reject) {
-        waitForDockerContainerToBeReady(dockerOpts, passError(reject, function () {
-            state.containerReference.getConString(databaseName, passError(reject, function (conString) {
+        if (process.env.GITLAB_CI) {
+            var pgUser = process.env.POSTGRES_USER;
+            var pgPassword = process.env.POSTGRES_PASSWORD;
+            var port = process.env.POSTGRES_PORT_5432_TCP_PORT;
+            var ip = process.env.POSTGRES_PORT_5432_TCP_ADDR;
```

Comment:
It was intentional to make it specific for gitlab, it is supposed to run with a provisioned `postgres` service and those variables are exposed by default in the gitlab enviornment. 
I could grab the `POSTGRES_PORT` variable process it to get the ip and the port if that makes more sense. You can see a test run here https://gitlab.one.com/webdev-cph/taskmate/builds/13153